kombu/transport/redis.py

```diff
@@ -1341,7 +1341,6 @@ class SentinelManagedSSLConnection(
         SSL Connection.
         """
 
-        pass
```

Comment:
`kombu/transport/redis.py:1346:1: E303 too many blank lines (3)`

Reply:
https://results.pre-commit.ci/run/github/735498/1672162368.nnwd2OjEQLm19RjMnRcsjg

Reply:
I'll start working through this now :)

Thanks for pointing out.